apps/linode-marketplace-wazuh/provision.yml

```diff
@@ -0,0 +1,23 @@
+# wazuh
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+  # generate creds
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
+      
+    - name: generating wazuh_password
+      set_fact:
+        wazuh_password: "{{ lookup('password', '/dev/null length=25 chars=ascii_letters,digits') }}"
+
+    - name: writing wazuh_password to vars
+      lineinfile:
+        insertafter: EOF
+        path: group_vars/linode/vars
+        line: | 
+          wazuh_password: {{ wazuh_password }}
```

Comment:
```suggestion
    - name: writing wazuh_password to vars
      lineinfile:
        insertafter: EOF
        path: group_vars/linode/vars
        line: | 
          wazuh_password: {{ lookup('password', '/dev/null length=25 chars=ascii_letters,digits') }}
```

Reply:
we can write the creds directly to the vars file

Reply:
Accepted change.